GCMessengerSDKSample/GCMessengerSDKSample/ChatWrapperViewController.swift

```diff
@@ -140,14 +146,8 @@ extension ChatWrapperViewController: ChatControllerDelegate {
             print("started")
             stopSpinner(activityView: chatViewControllerActivityView)
         case .chatDisconnected:
-            var delayForPopupDisplay = 0.0
```

Comment:
This part was added, because if the popup appeared without delay, it caused a lot of automation E2E failures. Why was it removed?